doc/releases/v0.12.0.txt

```diff
@@ -18,13 +18,17 @@ v0.12.0 (Unreleased)
 
 - |Enhancement| In :func:`histplot`, added `stat="percent"` as an option for normalization such that bar heights sum to 100 (:pr:`2461`).
 
+- |Enhancement| |Fix| Improved integration with the matplotlib color cycle in most axes-level functions (:pr:`2449`).
```

Comment:
This note is not relevant to this PR; I think I was tying off loose ends from a previous PR.